core/services/ocr2/plugins/promwrapper/plugin.go

```diff
@@ -0,0 +1,141 @@
+package promwrapper
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+)
+
+var _ types.ReportingPlugin = &promPlugin{}
+
+var (
+	promQuery = promauto.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "ocr2_reporting_plugin_query_duration",
+			Help:    "The amount of time elapsed during the OCR2 plugin's Query() method",
+			Buckets: []float64{}, // TODO: figure out buckets
+		},
+		[]string{"evmChainID", "pluginName"},
```

Comment:
nit/ does the `Name` suffix here add value? Or could they be just `plugin`?
```suggestion
		[]string{"evmChainID", "plugin"},
```

Reply:
since the object is called `promPlugin`, I can just name the field as `name` instead of `pluginName`. `plugin` will be used to refer to the underlying `wrapped` object.